index.html

```diff
@@ -3,60 +3,70 @@
 <head>
     <title>juicy-select</title>
 	<meta charset="utf-8" />
-    <script type="text/javascript" src="../webcomponentsjs/webcomponents.js"></script>
+    <script type="text/javascript" src="../webcomponentsjs/webcomponents-hi.js"></script>
```

Comment:
This will only work in Chrome and Safari Tech Preview. For supporting all browsers use `webcomponents-lite.js`. See: https://github.com/webcomponents/webcomponentsjs